bench/benchmarks/map_load.js

```diff
@@ -10,6 +10,9 @@ module.exports = class MapLoad extends Benchmark {
                 sources: {},
                 layers: []
             }
-        }).then(map => map.remove());
+        }).then(map => map.remove())
+            .catch(error => {
```

Comment:
Let's put the error handling in the `Benchmark` base class, wherever any of the overridable methods (`setup`, `bench`, and `teardown`) are called. That way it's consistent for all benchmarks.